crates/ty_python_semantic/src/semantic_index/definition.rs

```diff
@@ -334,14 +334,14 @@ pub(crate) struct MatchPatternDefinitionNodeRef<'ast> {
 
 impl<'db> DefinitionNodeRef<'_, 'db> {
     #[expect(unsafe_code)]
-    pub(super) unsafe fn into_owned(self, parsed: ParsedModuleRef) -> DefinitionKind<'db> {
+    pub(super) unsafe fn into_owned(self, parsed: &ParsedModuleRef) -> DefinitionKind<'db> {
```

Comment:
Can we remove the `unsafe` from this method, given that it's no longer UB if `node` doesn't belong to `parsed`?